### ISSUE TITLE
Swappable + chainable states of static panels above clearing panels

### ISSUE THREAD

**Issue by conormackeigan** [omenking/swap-n-pop#227]
So in Tetris Attack, when panels are clearing, the panels above them are swappable until the frame the final clearing panel becomes invisible. At that point the panels above are put into hang state, preventing them from clearing if a match is made, and preventing them from being swapped. The four-frame swap time can be abused in conjunction with the hanging state to achieve chains like this:

![swap-hover-clear-ta](https://user-images.githubusercontent.com/15354263/34428895-59a7f1fe-ec1f-11e7-94d2-9f94942d63c8.gif)

There are a few issues in Swap'N'Pop. Swapping panels are always set to STATIC state when the swap finishes, allowing them to clear, AND be swapped again within a one-frame window. As a result, the swap-time-abuse chain in TA does not work:

![swap-hover-clear](https://user-images.githubusercontent.com/15354263/34428915-8dfdaaf2-ec1f-11e7-91f0-8c5f1beaa133.gif)

Additionally, since panels are in STATIC state for a single frame, they can be swapped back and forth infinitely, preventing the panels from ever falling:

![2017-12-28_22-32-40](https://user-images.githubusercontent.com/15354263/34428923-a6bc30e0-ec1f-11e7-9191-d03bbf0164d0.gif)

TL;DR panels being hard-set to STATIC state when completing a swap does not work, needs refactoring.

**Comment by conormackeigan** [comment]
Panels no longer clear when they're not supposed to, but a chain is not triggered on land

**Comment by conormackeigan** [comment]
Chain not triggering covered in #242
